source/common/router/config_impl.cc

```diff
@@ -371,6 +376,7 @@ void RouteEntryImplBase::finalizeRequestHeaders(Http::HeaderMap& headers,
   // Append user-specified request headers in the following order: route-level headers,
```

Comment:
Modify the comment to add "route action level" to the sequence (route-level headers, route-action-level, ...).